packages/fast-components-styles-msft/src/button/index.ts

```diff
@@ -254,15 +254,15 @@ const styles: ComponentStyles<ButtonClassNameContract, DesignSystem> = (
             "&, &:hover": {
                 color: outlineColor,
                 borderColor: outlineBorderColor,
-                ...applyTransaprentBackground(),
+                ...applyTransparentBackground(),
             },
             "&:focus": {
-                ...applyTransaprentBackground(),
+                ...applyTransparentBackground(),
                 borderColor: outlineBorderColor,
                 boxShadow: `inset 0 0 0 1px ${outlineBorderColor}`,

```

Comment:
@chrisdholt should we be using `ShadowConfig` here?

Reply:
Note: this was a "drive by" fixing of a typo in the button component.  Please open bugs for any other button issues. :-)